pages/landing/patcher/index.html

```diff
@@ -100,28 +100,28 @@
 
 features:
   - title: Update dependency versions
-    description: Like DependaBot, Patcher automatically updates you to the latest versions of dependencies
-    icon: /assets/img/landing-zone-features/features-7@3x.png
+    description: Patcher automatically updates you to the latest versions of dependencies
+    icon: '<i class="fa fa-refresh fa-3x" aria-hidden="true"></i>'
 
   - title: Handle breaking changes
-    description: Unlike DependaBot, Patcher can also patch your code to handle backwards incompatible changes
-    icon: /assets/img/landing-zone-features/features-1@3x.png
+    description: Patcher can also patch your code to handle backwards incompatible changes
+    icon: '<i class="fa fa-bolt fa-4x" aria-hidden="true"></i>'
```

Comment:
Note: you need to make the others 3x as well, not just the first!

Reply:
Oh, i did..... but forgot to push my change!

Reply:
thanks for noticing! pushing it now!